### DISCUSSION TITLE
How does the introduction of LLM "AI" (both in idea and execution) reconcile with MDN and Mozilla's stated ideals?

### DISCUSSION THREAD

**Discussion by Sephsekla** [mdn/mdn-community#421] [Announcements]
From the [Mozilla Manifesto](https://www.mozilla.org/en-US/about/manifesto/):

> **Principle 8**
> 
> Transparent community-based processes promote participation, accountability and trust.

There was no transparency, discussion or oversight to the introduction of this controversial technology, and indeed it appeared to [bypass standard procedures for contribution](https://github.com/mdn/yari/issues/9208#issuecomment-1615411943). What is the reason for this direct contradiction of a principle?

From the [About MDN page](https://developer.mozilla.org/en-US/about):

> **Accurate and vetted for quality**

> Through our GitHub documentation repository, contributors can make changes, submit pull requests, have their contributions reviewed and then merged with existing content. Through [this workflow](https://mdn-contributor-docs.mozilla.org/), we welcome the vast knowledge and experience of our developer community while maintaining a high level of quality, accurate content.

There are numerous examples in https://github.com/mdn/yari/issues/9208 of clearly incorrect, unvetted (by virtue of its generation) content being displayed to visitors to the site. If this LLM-based content continues to disprove this claim, does the MDN team plan to remove this section from the `About` page, since it is now provably untrue?

**Comment by caugner** [comment]
We recorded your question as number 40.

**Comment by Ultrabenosaurus** [comment]
@LeoMcA @caugner @Rumyra have MDN abandoned their promise to answer all community questions?
